Sources/HaCWebsiteLib/Controllers/LandingPageController.swift

```diff
@@ -50,4 +36,30 @@ struct LandingPageController {
       Log.error("Socket error occured")
     }
   }
+}
+
+extension Event {
+  func toPostCard() -> PostCard? {
+    switch self {
+      case is WorkshopEvent:
+        return PostCard(
+          title: self.title,
+          category: .workshop,
+          description: self.eventDescription,
+          backgroundColor: self.colour, //TODO
+          imageURL: self.imageURL
+        )
+      case is HackathonEvent:
+        return PostCard(
+          title: self.title,
+          category: .hackathon,
+          description: self.eventDescription,
+          backgroundColor: self.colour, //TODO
+          imageURL: self.imageURL
+        )
+      default:
+        return nil
```

Comment:
:+1: as this allows us (for whatever reason) to have events that don't make sense to be rendered in a post card.